.eslintrc.json

```diff
@@ -0,0 +1,14 @@
+{
+  "env": {
+    "node": true,
+    "mocha": true
+  },
+  "extends": "eslint:recommended",
```

Comment:
Wasn't the project using the Standard.js ruleset?

Reply:
No the linting config was never setup when someone added all the eslint dependencies.

This config is a base which checks for known problems without enforcing a style.

A style can be added to eslint in further work, at least in this PR eslint actually runs from CI and spots problems (like the ones above).